server/src/main/java/org/apache/druid/server/coordinator/BalancerStrategy.java

```diff
@@ -46,13 +46,18 @@
   ServerHolder findNewSegmentHomeBalancer(DataSegment proposalSegment, List<ServerHolder> serverHolders);
 
   /**
-   * Find the best server on which to place a {@link DataSegment} replica according to the balancing strategy
+   * Finds the best servers on which to place a replica of the {@code proposalSegment}
+   * according to the balancing strategy.
```

Comment:
Okay, I will try to update the javadoc to reflect this distinction. But, as it happens, the same method is used for both "new/unavailable" segments as well "replicating/available" segments. (this would go away if we do a round-robin assignment in the first cycle for every segment) 
So I guess the method name should just be `findSegmentHomes`?

Reply:
Updated.